reference/best_practices/reusable_bundles.rst

```diff
@@ -1,4 +1,164 @@
 Reusable Bundles for Akeneo PIM community
 =========================================
 
-Coming soon!!! :)
+You are an integrator and you want to share reusable parts of your project with the awesome Akeneo PIM community?
+Let's read our pieces of advice to write a great and documented bundle!
+
+
+What is the purpose of a reusable bundle?
+
+* **Compatibility**: to fit Akeneo PIM features
+* **Maintainability**: to make your code flexible and testable (and tested)
+* **Extensibility**: to allow integrators to use your bundle to fit with their specific needs
+* **Scalability**: to know your application limits and document them
+
+
+Compatibility
+-------------
+
+Your bundle won't be reusable if you forget some cases. That is no more an issue if you document compatibilities of your bundle.
+
+Akeneo PIM storage stack comes with MySQL and hybrid MySQL/MongoDB (:doc:`/reference/technical_information/choose_database`).
+You can refer to the internal API to be sure you develop your bundle in the right way.
+Follow these cookbooks for more details: ":doc:`/cookbook/catalog/product/index`" and ":doc:`/cookbook/catalog/common/index`".
+
+There are two existing distributions CE/EE. The main project is `Akeneo PIM Community Edition`_ (CE) but
+we also have an `Enterprise Edition`_ (EE) which comes with several of extra features dedicated to advanced needs.
+
+Many attribute types are managed (14 in CE and one more in EE). Make sure you do not forget any use case for an attribute type.
+
+Try your bundle with other catalogs. Don't forget use cases with many locales, many channels and various catalog volumes.
+You can use our `data generator bundle`_ to generate a catalog with random data.
+
+.. _Akeneo PIM Community Edition: https://github.com/akeneo/pim-community-standard
+.. _Enterprise Edition: https://www.akeneo.com/enterprise-edition/
+.. _data generator bundle: https://github.com/akeneo-labs/DataGeneratorBundle
+
+
+Maintainability
+---------------
+
+Keep in mind the `Single Responsibility principle`_ from the five `SOLID principles`_ of Object Oriented Programming.
```

Comment:
Keep in mind the five `SOLID principles`_ of Object Oriented Programming and especially the `Single Responsibility principle`_.